### PR TITLE
Deepmd potential file, training set and validation set added

### PR DESCRIPTION
**Pull request by 24ssuman**

### CODE REVIEW

**Review by Miniland1333 — CHANGES_REQUESTED** [review]
Please add citation information about your publication to `README.md` and  `DC-r2SCAN_water_autoionization_nanoconfinement/readme.md`

**Review by Miniland1333 — APPROVED** [review]
LGTM
